graf2d/gpad/inc/TRatioPlot.h

```diff
@@ -135,59 +136,60 @@ class TRatioPlot : public TPad {
    virtual void SetupPads();
    virtual void CreateVisualAxes();
    virtual Bool_t SyncPadMargins();
-   virtual void SetPadMargins();
-   virtual void CreateGridline();
+   void SetPadMargins();
+   void CreateGridline();
 
-   virtual void ImportAxisAttributes(TGaxis* gaxis, TAxis* axis);
+   void ImportAxisAttributes(TGaxis* gaxis, TAxis* axis);
 
-   virtual Bool_t IsDrawn();
+   Bool_t IsDrawn();
 
-   virtual void Init(TH1* h1, TH1* h2, Option_t *option = "", Option_t *h1DrawOpt = "hist", Option_t *h2DrawOpt = "E", Option_t *graphDrawOpt = "AP");
+   virtual void Init(TH1* h1, TH1* h2, Option_t *option = "");
 
 public:
 
    TRatioPlot();
    virtual ~TRatioPlot();
-   TRatioPlot(TH1* h1, TH1* h2, Option_t *option = "pois", Option_t *h1DrawOpt = "hist", Option_t *h2DrawOpt = "E", Option_t *graphDrawOpt = "AP", const char *name = "RatioPlot", const char *title = "RatioPlot", Double_t xlow = 0, Double_t ylow = 0, Double_t xup = 1, Double_t yup = 1);
-
-   TRatioPlot(THStack* st, TH1* h2, Option_t *option = "pois", Option_t *h1DrawOpt = "hist", Option_t *h2DrawOpt = "E", Option_t *graphDrawOpt = "AP", const char *name = "RatioPlot", const char *title = "RatioPlot", Double_t xlow = 0, Double_t ylow = 0, Double_t xup = 1, Double_t yup = 1);
+   TRatioPlot(TH1* h1, TH1* h2, Option_t *option = "pois");
 
-   TRatioPlot(TH1* h1, Option_t *option = "", Option_t *h1DrawOpt = "", Option_t *graphDrawOpt = "LX", TFitResult *fitres = 0, const char *name = "RatioPlot", const char *title = "RatioPlot", Double_t xlow = 0, Double_t ylow = 0, Double_t xup = 1, Double_t yup = 1);
+   TRatioPlot(THStack* st, TH1* h2, Option_t *option = "pois");
 
+   TRatioPlot(TH1* h1, Option_t *option = "", TFitResult *fitres = 0);
 
+   void SetH1DrawOpt(Option_t *opt);
+   void SetH2DrawOpt(Option_t *opt);
+   void SetGraphDrawOpt(Option_t *opt);
 
    virtual void Draw(Option_t *chopt="");
    virtual void Browse(TBrowser *b);
 
    virtual void BuildLowerPlot();
```

Comment:
1) Is this a function that users should be able to call?
2) It's marked virtual, so I should be able to re-implement it in a derived class. But that derived class cannot access the private data members - so how can that derived class's reimplementation do anything useful? Maybe mark non-virtual; or make this return a `TGraph*` (or a `TObject*`?), or make the data members protected.